src/providers/MeetingEventProvider/index.tsx

```diff
@@ -6,20 +6,27 @@ import { EventName, EventAttributes } from 'amazon-chime-sdk-js';
 
 import { useMeetingManager } from '../MeetingProvider';
 
-type MeetingEventProviderContextType = {
-  name: EventName;
-  attributes: EventAttributes;
-} | undefined;
+type MeetingEventProviderContextType =
+  | {
```

Comment:
Thischange is made by prettier auto fix . I am not sure it is just some styling changes enforced by prettier if i remove it. Prettier gives this error " Insert `⏎··|` "

Reply:
It's a styling change when you define an object property in the following line.

Applying Prettier to the following code won't change because `{ name: EventName` is in the same line.
```
type MeetingEventProviderContextType = { name: EventName; attributes: EventAttributes } | undefined;
```

But the following code will start with |.
```
type MeetingEventProviderContextType = {
  name: EventName; attributes: EventAttributes } | undefined;
```